lib/cucumber/events/step_definition_registered.rb

```diff
@@ -14,6 +14,8 @@ class StepDefinitionRegistered < Core::Event.new(:step_definition)
       # _@private
       def initialize(step_definition)
         @step_definition = step_definition
+
+        super
```

Comment:
Same here

Reply:
Same response.

But yeah, trying to cover that would be helpful